tools/doc/node_modules/js-yaml/index.js

```diff
@@ -1,15 +1,7 @@
 'use strict';
 
-// Hack to load the js-yaml module from eslint.
-// No other reason than that it’s huge.
 
-const path = require('path');
+var yaml = require('./lib/js-yaml.js');
 
-const realJSYaml = path.resolve(
-  __dirname, '..', '..', '..', // tools/
-  'eslint',
-  'node_modules',
-  'js-yaml'
-);
 
-module.exports = require(realJSYaml);
+module.exports = yaml;
```

Comment:
Stray change.

Reply:
This is happening when you run `make doc` … we really should get rid of these `npm install`s :(

Reply:
Or `.gitignore` them, everyone making a PR yesterday ran into this

Reply:
If I'm not totally mistaken, adding an already tracked file to `.gitignore`  has no effect 😞 . Something like `git update-index --assume-unchanged` would work, but every developer has to run that locally, not really a solution for this problem.